data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,736 @@
+			`	Sayari gives a look of concern. "Are you suggesting that this captain was a criminal?"`
+			`	"Highly likely," Xilin responds. "A pirate with a stolen ship and a counterfeit license."`
+			`	"I will inform our spaceport authorities to look out for this captain and his ship," Sayari says. "It may also be beneficial if you were to make this database available to us, so that we may monitor other potential criminal humans."`
+			`	"Unfortunately, I'd have to gain authorization for that," says Xilin. "But I'll see what I can do."`
+			choice
+				`	"Should I help track down that captain?"`
+					goto track
+				`	"So are pirates trying to sneak their way into Hai territory?"`
+			`	"That may be the case," Xilin says. "We'd be stupid to suspect this as only having been an isolated incident."`
+				goto pirates
+			label track
+			`	"No, leave that to the Hai," Xilin says. "At least for so long as this is our only lead on pirate activity in Hai territory, although I imagine that we'll find more if we go looking for it."`
+			label pirates
+			`	"We have had run-ins with your pirates before," Sayari says. "Most of the time they cause only mischief. Very rarely do they raid our fleets or cause any large trouble. I fear though that this situation will only embolden them."`
+			`	Just then, Alondo pokes his head into the room looking somewhat exasperated. "Captain <last>. Could you give Remington a hand? He's out on the landing pads." He disappears again without waiting for a reply, and you hear his heavy footsteps grow quieter as he walks away.`
+			`	"You go help Remington," Xilin says. "I'll continue my work here."`
+
+
+
+mission "Hai Reveal: Meet the Team: Samuel"
+	name "High Tension"
+	description "Remington has heard news of tensions between new arrivals and local Hai on <stopovers>. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Cloudfire"
+	passengers 2
+	blocked "You will need <capacity> in order to take this mission."
+	to offer
+		has "hr: meet the team"
+		has "Hai Reveal: Meet the Team: Xilin: done"
+	on offer
+		log "People" "Terry Adrianopoulos" `Terry Adrianopoulos is the secretary of Samuel Remington, or at least that's how he puts it. Remington mentioned an internal political situation within the Syndicate while glancing at Terry, so there may be something larger at play with her presence. Hopefully it doesn't conflict with resolving the situation with the Hai.`
+		conversation
+			`You exit the building onto the landing pads with Sayari following behind you and find the tall man and the woman from the front row of the briefing waiting by your ship.`
+			branch met
+				has "met remington"
+			action
+				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai.`
```

Comment:
Putting the internal Syndicate conflict in Sam's entries, since it's still pretty important.
```suggestion
				log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations. He is acting as the Syndicate's representative in helping to deal with the situation regarding the Hai, and thinks the role will be a good break from the Syndicate's current internal turmoil.`
```